octue/runner.py

```diff
@@ -57,15 +57,13 @@ def __init__(
         self.output_manifest_path = output_manifest_path
         self.children = children
         self.skip_checks = skip_checks
-
-        # Store the log level (same log level used for all analyses)
         self._log_level = log_level
         self.handler = handler
 
         if show_twined_logs:
             apply_log_handler(logger=package_logger, handler=self.handler)
 
-        # Ensure the twine is present and instantiate it
+        # Ensure the twine is present and instantiate it.
```

Comment:
I prefer to leave full stops off unless it's a multi-sentence comment. Happy to change but we should decide on a style and note it.

Reply:
Ah cool. I think it's more consistent to end all comments with full stops? Although in docstring `:param` descriptions, I prefer to leave them off for some reason?